config.defaults.yaml

```diff
@@ -0,0 +1,51 @@
+kowalski:
+  host: kowalski.caltech.edu
+  port: 443
+  protocol: https
+  token:
+  collection_sources: ZTF_sources_20201201
+taxonomy:
```

Comment:
We should have long_forms of the variability types somewhere (may be they are, I just have not seen them), and linked from somewhere close to here.

Reply:
I think we can work on that separately, could be a dedicated section in the field guide. Do you want to file an issue for that @AshishMahabal?